### PR TITLE
Migrated metrics.apib

### PR DESCRIPTION
**Pull request by avigoldman**

### CODE REVIEW

File: content/api/metrics.apib [hunk 1]
```diff
@@ -7,62 +7,69 @@ description: Query real-time data about message processing, message disposition,
 SparkPost logs copious amounts of statistical, real-time data about message processing, message
```
Comment (jgzamora) [inline content/api/metrics.apib]: "the app(link)"

File: content/api/metrics.apib [hunk 2]
```diff
@@ -7,62 +7,69 @@ description: Query real-time data about message processing, message disposition,
...
 disposition, and campaign performance.  This reporting data is available in the UI or through the Metrics API.  The Metrics API provides a variety of endpoints enabling you to retrieve a summary of the data, data grouped by a specific qualifier, or data by event type.  Within each endpoint, you can also apply various filters to drill down to the data for your specific reporting needs.
 
-<banner status="warning"><strong>Deprecation Notice</strong>: The <tt>bindings</tt> and <tt>binding_groups</tt> query parmeters have been deprecated. Please use the <tt>sending_ips</tt> and <tt>ip_pools</tt> query parameters instead.</banner>
+<banner status="warning"><strong>Deprecation Notice</strong>: The <code>bindings</code> and <code>binding_groups</code> query parmeters have been deprecated. Please use the <code>sending_ips</code> and <code>ip_pools</code> query parameters instead.</banner>
```
Comment (jgzamora) [inline content/api/metrics.apib]: Can we remove this from here and just leave the ones by the endpoints?

File: content/api/metrics.apib [hunk 3]
```diff
@@ -7,62 +7,69 @@ description: Query real-time data about message processing, message disposition,
...
+
+
+<banner status="info"><strong>Note</strong>: For a given request, average first attempt delivery latency can be calculated as <code>total_delivery_time_first / count_delivered</code>. A similar calculation holds for <code>total_delivery_time_subsequent</code>.</banner>
+
+
+### Discoverability Links [GET /metrics/]
+
+The Metrics API is designed for discoverability of child links. Calling the API root displays a list of URIs that exists within the Metrics API.
+
+<banner status="info"><strong><a href="https://www.sparkpost.com/enterprise-email/">SparkPost Enterprise customers only</a></strong>: Metric discoverability links for binding-groups and bindings apply to SparkPost Enterprise customers only.</banner>
```
Comment (jgzamora) [inline content/api/metrics.apib]: Did we reach any conclusion on whether Binding Groups are at all still in use in Enterprise?

File: content/api/metrics.apib [hunk 4]
```diff
@@ -421,200 +184,42 @@ aggregate data, which can be used as "group by" qualifiers.
...
+            ]
         }
-        ```
 
-### Deliverability Metrics by Domain [GET]
+
+
+### Metrics by Domain [GET /metrics/deliverability/domain{?from,to,delimiter,domains,campaigns,templates,sending_ips,ip_pools,sending_domains,subaccounts,metrics,timezone,order_by,limit}]
 
 Provides aggregate metrics grouped by domain over the time window specified.
```
Comment (jgzamora) [inline content/api/metrics.apib]: we should specify "recipient domain"

File: content/api/metrics.apib (hunk 3 above)
Comment (avigoldman) [reply content/api/metrics.apib]: It is a case-by-case basis so I'm going to leave this along for now.

**Pull request merged**